src/components/Quote.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+
+const Quote = () => {
+  const [quote, setQuote] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
```

Comment:
- [ ] The implementation of **Quote.js** is ok in accordance to the requirement 👍, but <Quotes> component should be imported into the main component as specifically mentioned in the requirement, kindly follow the procedure given. 👇 

<img width="496" alt="Untitled" src="https://github.com/tajemouti/math-magicians/assets/103386183/8f1f2ef2-a3dd-43e5-90c1-162daa297a3b">

Reply:
Hi again @OyePriscilla,

Upon revisiting the milestone requirements and carefully considering your feedback, there might be a slight misinterpretation of the requirement regarding the placement of the Quote component. The requirement states: "Import the new component into the main component (next to the Calculator). It should be on the same page as the Calculator."
In light of this, I made sure to adhere to the requirement by importing the Quote component into the main component, which is the 'App' component. This placement effectively fulfills the intent of having the Quote component on the same page as the Calculator.
To clarify, the main component that was referred to is 'App' and not the 'Calculator' component. By including the Quote component in 'App', I've met the requirement's intent of having both the Calculator and Quote components on the same page.
I apologize for the confusion, and I hope this explanation clarifies the implementation approach I've taken. Your feedback has been valuable, and I'm going to request a re-review with the requested change anyway, but I'm confident that my code is aligned with the milestone's requirements.
Thank you once again for your time and attention. I look forward to any further guidance or suggestions you might have.

Cheers